NEWS

```diff
@@ -1,3 +1,25 @@
+22-Sep-2017 IGNITION v0.19.0
+
+  Features
+
+    - Add support for cloudstack network metadata
+    - Add blackbox tests for tfpt urls
+    - Removed dependency on kpartx for blackbox tests
+
+  Changes
+
+    - Stop adding extra quotes around GECOS field when creating users
+    - Use a private tmpdir subdirectory hen running blackbox tests
```

Comment:
`when`, if this is worth release-noting at all.  It should only affect behavior when `TMPDIR` is set.